### ISSUE TITLE
Add arrow to Jira ticket link tooltip

### ISSUE THREAD

**Issue by raghavaggarwal2308** [mattermost/mattermost-plugin-jira#886]
This issue is a part of issue #774

We are supposed to add an arrow to the Jira ticket tooltip as shown in the screenshot below:
![image](https://user-images.githubusercontent.com/55234496/198960798-7a43bc4d-c03e-4562-a437-3376344b00e5.png)

The change to add the arrow is dependent on PR [#10235](https://github.com/mattermost/mattermost-webapp/pull/10235). So, this change was put on hold until the [PR](https://github.com/mattermost/mattermost-webapp/pull/10235) is merged into mattermost-webapp.

To understand this issue thoroughly read [this](https://github.com/mattermost/mattermost-plugin-jira/pull/835#issuecomment-1088767784) discussion.
